src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -167,7 +175,26 @@ const PaymentFormContent = ({ setConfirmationToken }) => {
       return
     }
 
-    setConfirmationToken({ confirmationToken: response.me?.confirmationToken })
+    const updateOrderPaymentMethodResult =
+      await updateOrderMutation.submitMutation({
+        variables: {
+          input: {
+            id: order.internalID,
+            paymentMethod: "CREDIT_CARD",
+          },
+        },
+      })
+
+    validateAndExtractOrderResponse(
+      updateOrderPaymentMethodResult.updateOrder?.orderOrError,
+    )
```

Comment:
Anywhere that we call this function could throw an error, so it might be good to put this in a try/catch. However we have more tickets to deal with error handling so maybe not blocking.

Reply:
For payment stuff, might as well get that try/catch done in this PR and make it blocking, because we don't want to be swallowing errors during dev